### DISCUSSION TITLE
How to manage tarball committed/forked in our repository?

### DISCUSSION THREAD

**Discussion by tai271828** [solvcon/devenv#30] [Ideas]
We may probably want to use tarball to build some packages that is more stable and/or less release frequency. Besides, `wget` or `curl` download each time, tools like `git submodule` may be also the solution for fetching a bunch of tarball.

Do we have other solutions, or any comments/suggestions for the implementation details?

Please follow the initial thread for more context https://github.com/solvcon/devenv/pull/29#discussion_r635640267

**Comment by yungyuc** [comment]
> We may probably want to use tarball to build some packages that is more stable and/or less release frequency. Besides, `wget` or `curl` download each time, tools like `git submodule` may be also the solution for fetching a bunch of tarball.
> 

Tarballs may be hashed and cached.  A download script may keep the hash and verify the archived tarballs.  It may not download the tarballs again as long as the hash adds up.

> Do we have other solutions, or any comments/suggestions for the implementation details?
> 
> Please follow the initial thread for more context [#29 (comment)](https://github.com/solvcon/devenv/pull/29#discussion_r635640267)

**Comment by tai271828** [comment]
https://github.com/solvcon/devenv/issues/32 is a issue ticket for the tarball management.
